policies/modules/ROOT/pages/policies-included-apply.adoc

```diff
@@ -5,9 +5,9 @@ endif::[]
 :keywords: policy, endpoint
 :page-aliases: api-manager::using-policies.adoc
 
-In Connected Mode and Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.
+For Flex Gateway running in Connected Mode or for Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.
 
-In Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
+For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Flex Gateway Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Comment:
```suggestion
For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Either rename that second mention of "Flex Gateway" so that it matches the naming of the instance in the previous sentence, or omit it.

We need to be consistent with how we refer to Flex Gateway running in the two modes.